run-ab-platform.sh

```diff
@@ -19,27 +19,28 @@ PS4="$blue_text""${BASH_SOURCE}:${LINENO}: ""$default_text"
 Help()
 {
    # Display Help
-   echo "This Script will download the necessary files for running docker compose"
-   echo "It will also run docker compose up"
-   echo "Take Warning! These assets may become stale over time!"
+   echo -e "This Script will download the necessary files for running docker compose"
+   echo -e "It will also run docker compose up"
+   echo -e "Take Warning! These assets may become stale over time!"
    echo
    # $0 is the currently running program
-   echo "Syntax: $0"
-   echo "options:"
-   echo "   -d --download    Only download files - don't run docker compose"
-   echo "   -r --refresh     ${red_text}DELETE${default_text} existing assets and re-download new ones"
-   echo "   -h --help        Print this Help."
-   echo "   -x --debug       Verbose mode."
-   echo
+   echo -e "Syntax: $0"
+   echo -e "options:"
+   echo -e "   -d --download    Only download files - don't run docker compose"
+   echo -e "   -r --refresh     ${red_text}DELETE${default_text} existing assets and re-download new ones"
+   echo -e "   -h --help        Print this Help."
+   echo -e "   -x --debug       Verbose mode."
+   echo -e ""
 }
 
 ########## Declare assets care about ##########
-docker_compose_yaml="docker-compose.yaml"
-            dot_env=".env"
-        dot_env_dev=".env.dev"
-              flags="flags.yml"
+      docker_compose_yaml="docker-compose.yaml"
+docker_compose_debug_yaml="docker-compose.debug.yaml"
```

Comment:
This is the main change.  everything else is just `echo -e`

Reply:
can you also add this to .gitignore?

Reply:
Disregard above, i see you did already